src/library/autodj/dlgautodj.cpp

```diff
@@ -372,6 +380,22 @@ void DlgAutoDJ::updateSelectionInfo() {
     }
 }
 
+void DlgAutoDJ::updateInfo() {
```

Comment:
maybe more specific?
```suggestion
void DlgAutoDJ::updateTotalInfo() {
```
nitpicking: it be consistent to name the labels accordingly, `labelSelectionInfo`, `labelTotalInfo`